app/app/src/main/kotlin/com/hedvig/app/feature/profile/ui/payment/PaymentDestination.kt

```diff
@@ -0,0 +1,321 @@
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.layout.widthIn
+import androidx.compose.material.Surface
+import androidx.compose.material.icons.Icons
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Divider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Switch
+import androidx.compose.material3.SwitchDefaults
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.hedvig.android.core.designsystem.component.button.HedvigContainedButton
+import com.hedvig.android.core.designsystem.component.button.HedvigContainedSmallButton
+import com.hedvig.android.core.designsystem.component.card.HedvigCard
+import com.hedvig.android.core.designsystem.component.textfield.HedvigTextField
+import com.hedvig.android.core.designsystem.material3.typeContainer
+import com.hedvig.android.core.designsystem.material3.typeElement
+import com.hedvig.android.core.designsystem.preview.HedvigPreview
+import com.hedvig.android.core.designsystem.theme.HedvigTheme
+import com.hedvig.android.core.icons.Hedvig
+import com.hedvig.android.core.icons.hedvig.normal.ChevronRight
+import com.hedvig.android.core.icons.hedvig.normal.Payments
+import com.hedvig.android.core.icons.hedvig.normal.Waiting
+import com.hedvig.android.core.ui.clearFocusOnTap
+import com.hedvig.android.core.ui.scaffold.HedvigScaffold
+import hedvig.resources.R
+import java.util.*
+import kotlinx.datetime.LocalDate
+
+@Composable
+fun PaymentDestination(navigateUp: () -> Unit) {
+
+}
+
+@Composable
+fun PaymentScreen(
+  uiState: PaymentUiState,
+  locale: Locale,
+  navigateUp: () -> Unit,
+) {
+  Box(
+    modifier = Modifier.fillMaxSize(),
+  ) {
+    HedvigScaffold(
+      topAppBarText = stringResource(R.string.PROFILE_PAYMENT_TITLE),
+      navigateUp = navigateUp,
+      modifier = Modifier.clearFocusOnTap(),
+    ) {
+      Column(modifier = Modifier.padding(horizontal = 16.dp)) {
+        Spacer(Modifier.height(16.dp))
+        NextPayment(uiState, locale)
+        Spacer(Modifier.height(4.dp))
+        Divider()
+        Spacer(Modifier.height(4.dp))
+        InsuranceCosts(uiState, locale)
+        Spacer(Modifier.height(4.dp))
+        TotalDiscount(uiState)
+        Spacer(Modifier.height(4.dp))
+        Divider()
+        Spacer(Modifier.height(16.dp))
+        AddDiscount(uiState)
+        Spacer(Modifier.height(32.dp))
+        PaymentDetails(uiState)
+        Spacer(Modifier.height(4.dp))
```

Comment:
This 4dp here is added because of the line 231 `Spacer(Modifier.height(4.dp))`, so it centers the item appropriately.
So now this screen is adding 4 dps under it, and the item itself is adding the 4dp over the item.
This would make it quite tricky to change if we ever re-order how these items come, or if the spacings change etc.

I'd suggest either:
* The item itself takes the space + 12dp vertical padding that it needs, and the screen up here does
```
Spacer(Modifier.height(4.dp))
PaymentDetails(uiState)
Spacer(Modifier.height(4.dp))
```
* The PaymentDetails item makes the vertical padding be 16dp directly, and the screen here does not add any special spacing around it
```
PaymentDetails(uiState)
```

<img width="487" alt="image" src="https://github.com/HedvigInsurance/android/assets/44558292/58fbb165-4c45-4fde-9dd5-dc3870b21578">

Considering the design here maybe in this case it's easier to just add the 16.dp paddings and then no need to add those 4.dp extra here nor in the component itself